docs.yaml

```diff
@@ -37,6 +37,9 @@ nanoc_config:
       exclude:
         - '^https?://travis-ci.org/*'
         - '^https?://ci.appveyor.com/*'
+        - 'https://twitter.com/datastax'
+        - 'https://academy.datastax.com/slack'
```

Comment:
Apologies for not capturing this earlier, but Academy Slack is also not in operation anymore. We need https://dtsx.io/discord instead.